mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -490,6 +490,8 @@ class PopulateParams : public PopulateParamsBase {
     {64, 32, 32, 4, 2, 2}};
   // clang-format on
 
+  InitParamsNonXDL ExtraParameters = {128, 64, 128, 4, 4, 4};
```

Comment:
@asleepzzz the purpose of this extra parameter needs to be documented. also i think `extraparameters` might not be a good name.